### ISSUE TITLE
Set content mode of background image

### ISSUE THREAD

**Issue by danielronnkvist** [tombenner/nui#364]
I'm setting a background image on the _NavigationBar_ but cannot solve how to set the content mode to _aspect fill_. Couldn't find any other information on another item, is it possible to add this feature? I think it would be appreciated on other items as well.

**Comment by Stunner** [comment]
Good idea. I unfortunately probably won't get to this any time soon. I would encourage you to make the change and send a pull request my way and I'll happily review it!
